conf.d/done.fish

```diff
@@ -163,21 +191,18 @@ if test -z "$SSH_CLIENT" # not over ssh
                 end
 
             else if uname -a | string match --quiet --regex Microsoft
-                set -l powershell_exe (command --search "powershell.exe")
-                if test $status -ne 0
-                    and command --search wslvar
-                    set -l powershell_exe (wslpath (wslvar windir)/System32/WindowsPowerShell/v1.0/powershell.exe)
+                if test "$__done_notify_sound" -eq 1
+                    set soundopt "-Sound Default"
+                else
+                    set soundopt "-Silent"
                 end
 
-                if string length --quiet "$powershell_exe"
-                    and test -x "$powershell_exe"
-                    and "$powershell_exe" -command "Import-Module -Name BurntToast" 2>/dev/null
-
-                    if test "$__done_notify_sound" -eq 1
-                        set soundopt "-Sound Default"
-                    end
-                    command "$powershell_exe" -command New-BurntToastNotification -Text \""$title"\",\""$message"\" $soundopt
-                end
+                __done_run_powershell_script '
+                    Import-Module -Name BurntToast 2>&1 | Out-Null
+                    if (Get-Module -Name BurntToast) {
+                        New-BurntToastNotification -Text "$title","$message" $soundopt
```

Comment:
Are these variables being expanded? AFAIK they won't be inside single quotes.

Reply:
You are right. Fixed that.